sources/packages/api/src-sql/RawSQL/FederalRestrictions/Bulk-update-active-restrictions.sql

```diff
@@ -0,0 +1,26 @@
+/*
+ * Update all active federal restrictions on table sims.student_restrictions
+ * that are still present of federal restrictions. This update is not
+ * critical but it will keep the created date and updated date representing
+ * in a more accurate way when the system received the restriction for the first
+ * time till when the same status was kept in the federal snapshot day after day.
+ */
+UPDATE

```

Comment:
Is this update really required? Asking for clarification because, on federal restriction daily file, a restriction is repeatedly sent to mention that restriction still exist and not resolved, so functionally there is no update to the restriction. Am i missing something?

Reply:
This update is not critical but keep in mind the update is happening on the student restrictions. Update the active restriction will give the system the ability to keep track of the last time that the restriction was received in the daily report from the federal file.
Consider the below scenario:
1 - New restriction is received on day 2021-12-01 -> (CreatedDate: 2021-12-01, Update Date: 2021-12-01)
2 - Same restrictions is still active on day next day -> (CreatedDate: 2021-12-01, Update Date: 2021-12-**02**) 
3 - Same restrictions is still active on day next day -> (CreatedDate: 2021-12-01, Update Date: 2021-12-**03**) 
So, IMO, it gives a better view that the restriction was created and the system is doing a follow-up on this while it is active. Does it make sense? So far, there is no reason to believe that this will bring a performance impact but is mentioned in the comments that it is not critical, so it could be removed in the future if there is some struggle, what I do not have reasons to believe right now that would be the case.
Please le me know if we need further discussions on this 😉

Reply:
It makes sense when it is helpful to find that a restriction which is active has to be updated within last 24 hours otherwise the nightly job isn't updated the records correctly.